sunflower-farmers/sunflower-farmers
# Audio player

This PR will add an Audio Player with the default song.
I paired with RickFurt on this one as he wants to start contributing to the project.

Should we release this before or with the BETA release?

![player](https://user-images.githubusercontent.com/25412194/143032206-020cba7c-41c3-4258-9ea9-9d7bb51dc2cf.gif)

### To Test: 
* hover over the musical note
* See the player showing
* test controls (Note the `nextTrack` button won't work as we only have one song at the moment)
